src/common/mfu_flist_copy.c

```diff
@@ -443,6 +447,37 @@ static int mfu_copy_xattrs(
     mfu_free(&list);
     list_bufsize = 0;
 
+    /* open the src */
```

Comment:
style: this new code should be moved to a helper function. It will allow the code to be a little simpler, exiting early if one of the syscalls fails, and will simplify reading mfu_copy_xattrs() when someone is looking for a bug unrelated to the FSGETXATTR/FSSETXATTR code.

Reply:
@KoyamaSohei please move the entire new block of code you're adding to a helper function.  It looks to me like you just need to pass in the source and destination path, and return an `int` indicating success or failure.